src/main/kotlin/racingcar/Racing.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar
+
+class Racing(private val cars: Cars) {
+
+    private val racing = mutableListOf<Cars>()
+
+    fun start(roundCount: Int, moveStrategy: MoveStrategy) {
+        for (j in 0 until roundCount) {
+            val racingCars = if (racing.isNotEmpty()) racing.last() else cars
+            racing.add(racingCars.move(moveStrategy))
+        }
```

Comment:
```suggestion
       repeat(roundCount) {
            val racingCars = if (racing.isNotEmpty()) racing.last() else cars
            racing.add(racingCars.move(moveStrategy))
        }
```

`repeat()`를 사용하면 반복에 대해 더 쉽게 처리할 수 있습니다  😊

참고
- [repeat()](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/repeat.html)